src/test/java/com/systelab/seed/patient/PatientResourceTest.java

```diff
@@ -106,6 +137,7 @@ public void testGetPatientList() {
         long initialSize = patientsBefore.getTotalElements();
         savePatientsDatabase(patientsBefore.getContent());
         TestUtil.checkANumber("List size", numberOfPatients, initialSize);
+        // TODO: Verify the elements in the list also.
```

Comment:
Remove the TODO: comment.

Reply:
I would not document any expected result for the method createSomePatients as it is an auxiliar method